.github/workflows/markdownlint-problem-matcher.json

```
@@ -0,0 +1,17 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "markdownlint",
```

```suggestion
      "owner": "markdownlint",
      "severity": "warning",
```
Can we use severity 'warning' to make these non blocking errors?

---

No, it maps to a regex https://github.com/actions/toolkit/blob/main/docs/problem-matchers.md

---

My idea is that `(yarn lint:md || exit 0)` will make the PR not fail but the problem mapper will annotate files with warnings(instead of errors).